src/main/java/at/hannibal2/skyhanni/config/features/garden/visitor/VisitorConfig.java

```diff
@@ -37,6 +37,12 @@ public class VisitorConfig {
     @FeatureToggle
     public boolean notificationChat = true;
 
+    @Expose
+    @ConfigOption(name = "Compact Visitor Chat", desc = "Compact reward summary messages when you accept a visitor's offer.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean compactVisitorRewardChat = false;
```

Comment:
remove the word "visitor" from that name. This setting is already inside the visitor category.